.github/workflows/release-please.yml

```diff
@@ -0,0 +1,23 @@
+on:
+    push:
```

Comment:
Are we doing this on a push to main - or are we going to start with a weekly manual release - sorry if I've misunderstood - will this make ever merge to main a new version?

Reply:
> are we going to start with a weekly manual release?

So the github action will create a release PR and we have full control over when that gets merged. A new release will only be published when we merge that release PR. We can decide how often we do that, it could be weekly, but we could choose to release more frequently (Bhavana made a good point [here](https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/pull/805#discussion_r1590277281) that releasing little and often is often preferred since it decreases risk).

> will this make every merge to main a new version?

No it won't, if there's no existing release PR then one will be created. With every subsequent merge to main, that release PR will get updated and the things included in the release will accumulate until we decide to publish the release (i.e. merge the release PR). 

My suggestion would be that if this gets merged today or Monday, we can try out a release that same day. Then I could show it in the sprint demo and if there's any PRs queued up we could go through publishing a release on Wed?

Reply:
I think we should discuss this as a team since there seem to be differing opinions here. I want to ask - why is it a bad thing to release every commit? I think the norm should be to release every commit as a new version, unless we choose not to for a specific reason (e.g., dependabot upgrades). 

We don't want to sit on top of changes, collecting them, unless we have a good reason for it. Thoughts from the team?

Reply:
Dependabot is the only thing that comes to mind as to why we wouldn't want to release every change